internal/models/profile.go

```diff
@@ -0,0 +1,52 @@
+package models
+
+import (
+	"github.com/CollActionteam/collaction_backend/internal/constants"
+	validation "github.com/go-ozzo/ozzo-validation"
+)
+
+type Profile struct {
+	UserID      string `json:"userid,omitempty"`
```

Comment:
I don't think these fields should be `omitempty`, because semantically even an empty bio does not change the fact that this profile has a bio section.

Reply:
Thanks, I will remove from others and leave only that on UserID and Phone, as I want to exclude them from the GetProfile response.